src/_baseMenu.js

```diff
@@ -889,28 +889,32 @@ class BaseMenu {
   /**
    * Handles click events throughout the menu for proper use.
    *
-   * Depending on what is supported either `touchstart` and `touchend` or
+   * Depending on what is supported `pointerdown` and `pointerup`, otherwise either `touchstart` and `touchend` or
    * `mousedown` and `mouseup` will be used for all "click" event handling.
    *
-   * - Adds a `touchend`/`mouseup` listener to the document so if the user clicks
+   * - Adds a `pointerup`/`touchend`/`mouseup` listener to the document so if the user clicks
    *   outside of the menu when it is open, the menu will close.
-   * - Adds a `touchstart`/`mousedown` listener to every menu item that will blur
+   * - Adds a `pointerdown`/`touchstart`/`mousedown` listener to every menu item that will blur
    *   all menu items in the entire menu structure (starting at the root menu) and
    *   then properly focus the clicked item.
-   * - Adds a `touchend`/`mouseup` listener to every submenu item that will properly
+   * - Adds a `pointerup`/`touchend`/`mouseup` listener to every submenu item that will properly
    *   toggle the submenu open/closed.
-   * - Adds a `touchend`/`mouseup` listener to the menu's controller
+   * - Adds a `pointerup`/`touchend`/`mouseup` listener to the menu's controller
    *   (if the menu is the root menu) so when it is clicked it will properly
    *   toggle open/closed.
    *
    * @protected
    */
   _handleClick() {
     // Use touch over mouse events when supported.
-    const startEventType = isEventSupported("touchstart", this.dom.menu)
+    const startEventType = isEventSupported("pointerdown", this.dom.menu)
```

Comment:
Given that all supported browsers support `pointerup`/`pointerdown` events we should clean this section up to only use the pointer events, this will get rid of the need for the event checks.

Doc comment will need to be updated as well.

Reply:
I kept the fallback to the original approach because I was not sure whether you just consider the last 2 versions of the supported browsers.

I can amend it if you confirm that is unnecessary to support old browser versions.

Reply:
I _believe_ we're in the clear for all the browsers that are supported. Chrome/FF/Edge have had full support for pointer events for a while now and Safari has had support since 2 versions ago.

Reply:
I've removed the event checks and updated the comment. Please review the changes

Reply:
Menu toggles and the `aria-expanded` attribute changes when I tested it in real browsers.

I think the test failed because JSDOM hasn't support PointEvent yet.

[PointerEvent constructor not available in window · Issue #2527 · jsdom/jsdom](https://github.com/jsdom/jsdom/issues/2527)